evm_loader/program/src/entrypoint.rs

```diff
@@ -267,14 +265,17 @@ fn process_instruction<'a>(
             let sysvar_info = next_account_info(account_info_iter)?;
             let operator_sol_info = next_account_info(account_info_iter)?;
             let collateral_pool_sol_info = next_account_info(account_info_iter)?;
-            let _user_eth_info = next_account_info(account_info_iter)?;
-            let _operator_eth_info = next_account_info(account_info_iter)?;
+            let operator_eth_info = next_account_info(account_info_iter)?;
+            let user_eth_info = next_account_info(account_info_iter)?;
             let system_info = next_account_info(account_info_iter)?;
 
             let _program_info = next_account_info(account_info_iter)?;
+            let _program_token = next_account_info(account_info_iter)?;
             let _program_code = next_account_info(account_info_iter)?;
```

Comment:
`_program_code` may not present in the account list

Reply:
There should be tests for this case.